src/entity/posts/BriefPost.ts

```diff
@@ -11,4 +11,7 @@ export class BriefPost extends Post {
 
   @Column({ nullable: true })
   readTime?: number;
+
+  @Column({ type: 'text', array: true, default: [] })
+  collectionSources: string[];
```

Comment:
Reusing collections sources column because graphorm already knows how to query each source data